addons/event_sms/models/event_mail.py

```diff
@@ -28,6 +28,9 @@ class EventMailScheduler(models.Model):
     def _selection_template_model(self):
         return super(EventMailScheduler, self)._selection_template_model() + [('sms.template', 'SMS')]
 
+    def _selection_template_model_get_mapping(self):
+        return {**super(EventMailScheduler, self)._selection_template_model_get_mapping(), 'sms': 'sms.template'}
```

Comment:
```suggestion
    mapping = super()._selection_template_model_get_mapping()
    mapping.update({'sms': 'sms.template'})
    return mapping
```

Reply:
Hello @abz-odoo 
@tde-banana-odoo suggested me to do like this (check here - https://github.com/odoo/enterprise/pull/41322#discussion_r1227619454)
Should I make changes according to your suggested changes?

Reply:
Hello @sauc-odoo , it's his module, let's do as he suggested 🙂